TsdrApi.php

```diff
@@ -44,6 +48,7 @@ class TsdrApi
 	*/
 	public static function getApiForm()
 	{
+		$constant = 'constant';
```

Comment:
Can we not return the value of a constant without the constant() function?? We should be able to.  I think the problem is you didn't put **echo** in-front of it.

Lets drop that function and use **echo** if we can.

If not, then we'll just leave it as is.

Reply:
I removed $constant.  Probably a little less readable now but with the payoff that the constant can be changed should the api change.